src/backend/distributed/deparser/deparse_statistics_stmts.c

```diff
@@ -66,6 +79,20 @@ AppendCreateStatisticsStmt(StringInfo buf, CreateStatsStmt *stmt)
 }
 
 
+static void
+AppendDropStatisticsStmt(StringInfo buf, List *nameList, bool ifExists)
+{
+	appendStringInfoString(buf, "DROP STATISTICS ");
+
+	if (ifExists)
+	{
+		appendStringInfoString(buf, "IF EXISTS ");
+	}
+
+	appendStringInfo(buf, "%s;", NameListToQuotedString(nameList));
```

Comment:
It's not harmful, but it's unnecessary to include a `;` here. Functions that do that have it wrong. It effectively turns this string into two statements.